crates/dojo-erc/src/tests/erc1155_tests.cairo

```diff
@@ -126,7 +126,7 @@ fn test_balance_of_zero() {
 
 
 #[test]
-#[available_gas(20000000)]
+#[available_gas(30000000)]
```

Comment:
hm why does gas usage increase here and below?

Reply:
Because index was set by default, and it used gas. Reverted after disabling index by default